src/AccessibilityInsights.SharedUx/Settings/ConfigurationModel.cs

```diff
@@ -413,6 +414,15 @@ public bool IsUnderElementScope
             set => SetDataValue<bool>(keyIsUnderElementScope, value);
         }
 
+        /// <summary>
+        /// The release channel configured for this client
+        /// </summary>
+        public ReleaseChannel ReleaseChannel
```

Comment:
do we need to update any config version values because of this change?

Reply:
The config version is essentially a constant now that the settings are explicitly stored as a dictionary. We wanted to enable people to switch between channels (forward or backward) without having to reconfigure their settings. The config version would only need to change if some future config file moved away from the Dictionary